Src/DependencyCollector/Shared/Implementation/HttpCoreDiagnosticSourceListener.cs

```diff
@@ -300,6 +300,10 @@ private void InjectRequestHeaders(HttpRequestMessage request, string instrumenta
                             {
                                 HttpHeadersUtilities.SetRequestContextKeyValue(requestHeaders, RequestResponseHeaders.RequestContextCorrelationSourceKey, sourceApplicationId);
                             }
+                            else
```

Comment:
Yup. I agree with @nizarq that when there's something wrong, it is equally difficult, if not more, to just track it as `HTTP`. I think we agreed to go with it unless we got user feedback / push back.